chromium_src/services/network/public/cpp/features.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/services/network/public/cpp/features.cc"
+
+#include "base/feature_override.h"
+
+namespace network::features {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kBrowsingTopics, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
aren't we disabling these already by stubbing out the services/factories?

Reply:
The service side for Browsing Topics and Shared Storage are disabled in `components/privacy_sandbox/brave_privacy_sandbox_settings.cc`. These flags also disable the functionality in blink.

Reply:
link to the relevant blink code?

Reply:
Created a follow up issue https://github.com/brave/brave-browser/issues/44713